src/index.js

```diff
@@ -9,6 +9,8 @@ import configureStore from './redux/store/configureStore';
 const store = configureStore().store;
 const persistor = configureStore().persistor;
 
+document.body.style = 'background: #f0f0f0;';
```

Comment:
Should this be here?

Reply:
... I legit don't remember writing this lol... So no

Reply:
Okay actually it does need to be there... I will see if I can do this another way but for now it is required to change the background of the entire screen and not just the part the app fills

Reply:
Interesting. I wonder if this could be accomplished in the `/public` folder? Assuming this changes our `index.html`.